src/confluent_kafka/admin/_config.py

```diff
@@ -172,31 +176,33 @@ def __init__(self, restype, name,
         self.configs = described_configs
         self.error = error
 
-    def __repr__(self):
+    def __repr__(self) -> str:
         if self.error is not None:
             return "ConfigResource(%s,%s,%r)" % (self.restype, self.name, self.error)
         else:
             return "ConfigResource(%s,%s)" % (self.restype, self.name)
 
-    def __hash__(self):
+    def __hash__(self) -> int:
         return hash((self.restype, self.name))
 
-    def __lt__(self, other):
+    def __lt__(self, other: 'ConfigResource') -> bool:
```

Comment:
Enforcement here might be awkward, since technically `__lt__` should be handling non-type match argumentsand giving a default answer or raising is NotImplemented -- which it appears to not implement

Reply:
Good point. I think maybe I should add the type check here and in other places to follow the pattern below:
```
def __lt__(self, other: object) -> bool:
    if not isinstance(other, MyClass):
        return NotImplemented
    return self.attribute < other.attribute
```